test/Interop/Cxx/operators/member-inline-address-only.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
```

Comment:
Could you extend existing tests, `test/Interop/Cxx/operators/member-inline.swift` etc., to cover exactly the same cases as right now but using your new address-only type?